components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```
@@ -16870,6 +16998,63 @@ components:
+      properties:
+        isValid:
+          type: boolean
+          description: |
+            This attribute declares whether this definition is valid or not.
+            example: true
+        errorMessage:
+          type: string
+          description: |
+            This attribute declares the validation error message
+        content:
+            type: string
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)